core/cmd/dkg_configure_command.go

```diff
@@ -130,29 +131,29 @@ func (cli *Client) ConfigureDKGNode(c *clipkg.Context) (*SetupDKGNodePayload, er
 	offChainPublicKey := ocr2.OffchainPublicKey()
 	configPublicKey := ocr2.ConfigEncryptionPublicKey()
 
-	// Set up bootstrapper job if bootstrapper.
 	if c.Bool("isBootstrapper") {
-		err = setupBootstrapperJob(cli, c, app)
+		// Set up bootstrapper job if bootstrapper.
+		err = createBootstrapperJob(cli, c, app)
 		if err != nil {
 			return nil, cli.errorOut(err)
 		}
-	}
-
-	// Set up DKG job.
-	dkgTemplateArgs := &DKGTemplateArgs{
-		contractID:              c.String("contractID"),
-		ocrKeyBundleID:          ocr2.ID(),
-		p2pv2BootstrapperPeerID: peerID,
-		p2pv2BootstrapperPort:   c.String("bootstrapPort"),
-		transmitterID:           transmitterID,
-		chainID:                 c.Int64("chainID"),
-		EncryptionPublicKey:     dkgEncryptKey,
-		KeyID:                   keyID,
-		SigningPublicKey:        dkgSignKey,
-	}
-	err = createDKGJob(cli, c, app, *dkgTemplateArgs)
-	if err != nil {
-		return nil, cli.errorOut(err)
+	} else {
+		// Set up DKG job.
+		dkgTemplateArgs := &DKGTemplateArgs{
```

Comment:
Small nit: why not just create this struct on the stack (i.e without the `&`) since you dereference it on line 153 anyway? i.e

```golang
dkgTemplateArgs := DKGTemplateArgs{...}
err = createDKGJob(cli, c, app, dkgTemplateArgs)
```
It might even be cleaner if you just pass in the struct literal to the function call:

```golang
err = createDKGJob(cli, c, app, DKGTemplateArgs{
// .. fields ...
})
```